sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -511,16 +534,17 @@ public virtual async Task StopProcessingAsync(CancellationToken cancellationToke
                     ActiveReceiveTask.Dispose();
                     ActiveReceiveTask = null;
                 }
-                if (Receiver != null)
-                {
-                    await Receiver.CloseAsync().ConfigureAwait(false);
-                }
+            }
+            catch (Exception exception)
+            {
+                ServiceBusEventSource.Log.StopProcessingException(Identifier, exception);
+                throw;
             }
             finally
             {
                 ProcessingStartStopSemaphore.Release();
-                //Logger.EventProcessorStopComplete(Identifier);
             }
+            ServiceBusEventSource.Log.StopProcessingComplete(Identifier);
```

Comment:
I was intentionally leaving these out of the finally block since if an exception is thrown we probably don't want to log that the operation was completed.

Reply:
I'm fine either way but in Event Hubs we log this even if the exception thrown.

This log is just saying "This method has been completed". It is not saying "This method has been successfully completed".

But again no strong opinion :)

Reply:
Yeah, I guess I was going with how this was done in Track 1 SB. Also, don't really have a strong opinion on this, but I think it is fine to leave as is for now.